docs/experiments-doc2query.md

```diff
@@ -87,5 +87,60 @@ So, this simple trick improves MRR by a bit over baseline Doc2query.
 
 ## TREC CAR
 
-Instructions coming soon!
+We will now describe how to reproduce the TREC-CAR results of our model BM25+doc2query presented in the paper.
 
+To start, download the TREC-CAR dataset and the predicted queries (i.e., document expansions):
```

Comment:
Would it better to remove "i.e:, document expansions"? It may give the impression that the link contains the expanded documents?